hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntryPool.java

```diff
@@ -112,6 +112,10 @@ public BlockOutputStreamEntryPool(
                 .createByteBufferConversion(unsafeByteBufferConversion));
   }
 
+  ExcludeList createExcludeList() {
+    return new ExcludeList();
```

Comment:
Do we need to use the constructor `public ExcludeList(long autoExpiryTime, java.time.Clock clock)` here, so the clock instance gets setup in the exclude list?

Reply:
I see now, this is in a different place to another instance below in ECBlockOutputStreamEntryPool.java - are we not concerned about expiring excluded nodes in BlockOutputStreamEntryPool.java  ?